app/lib/frontend/templates.dart

```diff
@@ -473,14 +473,25 @@ class TemplateService {
     } else {
       pageDescription += ' Dart package';
     }
+    final pageTitle =
+        '${package.name} $versionString| ${isFlutterPackage ? 'Flutter' : 'Dart'} Package';
     pageDescription += ' - ${selectedVersion.ellipsizedDescription}';
-    return renderLayoutPage(PageType.package, content,
-        title:
-            '${package.name} $versionString| ${isFlutterPackage ? 'Flutter' : 'Dart'} Package',
-        pageDescription: pageDescription,
-        faviconUrl: isFlutterPackage ? staticUrls.flutterLogo32x32 : null,
-        canonicalUrl:
-            isVersionPage ? _canonicalUrlForPackage(package.name) : null);
+    final canonicalUrl =
+        isVersionPage ? _canonicalUrlForPackage(package.name) : null;
+    final headerHtml = _renderTemplate('pkg/meta_header', {
+      'title': pageTitle,
+      'description': pageDescription,
+      'canonicalUrl': canonicalUrl,
+    });
+    return renderLayoutPage(
+      PageType.package,
+      content,
+      title: pageTitle,
```

Comment:
I've started this with the concept of having different title and description here, but all the guides I check say that it is better if they are the same across the different metadata definitions. Shall I move these to the `layout.mustache` (now or in a follow-up)?

Reply:
Follow-up is fine. Keeping them the same is good...

Reply:
@isoos – should we have a bug open for the follow-up?

Reply:
Prepared it: https://github.com/dart-lang/pub-dartlang-dart/pull/907